sviluppo/lavorare-con-rails.md

```diff
@@ -1,23 +1,22 @@
-# Lavorare con Rails
+# Working with Rails
 
-## Seed
+## Seeds
 
-Oltre al readme, la presenza o l'assenza di buoni seed sono un altro degli elementi che determina
-quanto velocemente si può iniziare a lavorare su un progetto nuovo.
+In addition to the readme, the presence or absence of good seeds is another element that determines
+how quickly you can start work on a new project.
 
-Ogni volta che una nuova feature viene sviluppata, sarebbe buona norma scrivere dei seed che possono
-essere usati per testare la feature. Non è necessario essere troppo dettagliati e testare ogni
-possibile scenario, ma è utile permettere ai nuovi sviluppatori di farsi un'idea di come funzioni
-l'applicazione senza doverla configurare manualmente.
+Once a new feature has been developed, it's a good idea to write seeds that can be used to test it. 
+There is no need to be too detailed and test all possible scenarios, but it is useful to help new 
+developers have an idea how the application works without having to configure it manually.
 
-Di nuovo, per ispirazione sulla scrittura di seed si possono vedere alcuni dei progetti esistenti
-o chiedere in #sviluppo. 
+Again, for inspiration on how to write seeds, you can see some of the existing projects or ask in 
+\#sviluppo. 
```

Comment:
`\` ?

Reply:
@matteogalliani it's to escape the `#`, which is normally treated as a heading in Markdown when it's at the beginning of the line.

Reply:
cool, thanks